dev_docs/tutorials/performance/adding_custom_performance_metrics.mdx

```diff
@@ -404,6 +404,38 @@ An event using custom metrics will be indexed with the following structure:
 }
 ```
 
+#### Add inline documentation
+
+The description metadata field is allowing teams to document how a page is instrumentented
+
+```typescript
+...
+  onPageReady({
+    meta: {
+      rangeFrom,
+      rangeTo,
+      description: '[ttfmp_dependencies] Dependencies table is  ready after fetching top_dependencies.'
+    }
+  });
+...
+```
+
+#### Track TTFMP on refresh load
+
+TTFMP supports a new dimension: isInitialLoad, distinguishing between an initial page load and a refresh triggered by actions like the time picker.
+This requires instrumenting `onPageRefreshStart()` when refreshing the page.
+
+```typescript
+...
+  const { onPageRefreshStart } = usePerformanceContext();
+
+   const handleRefresh = () => {
+    onPageRefreshStart()
```

Comment:
What do you think about adding an image with the marker to explain what `onPageRefreshStart` does?

Reply:
Yes